rico.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: rico
+title: Rico
+order: '16'
+headerText:
+permalink: /rico/
+header:
+  image: rico
+  text: Rico by Karakun
+nav:
+  bottom: false
+---
+
+Implement applications while leveraging state-of-the art technologies, patterns and practices.
+
+## Overview
+
+Rico is an application framework that provides several functionalities and APIs to create enterprise applications. 
+This includes features like tracing or monitoring that are critical in developing applications, escpecially in modern cloud based and distributed application landscapes. 
+Thus, Rico provides generic APIs with specific implementation for the mostly used frameworks and toolkits in modern application development.
+
+Rico originates from the _Canoo Dolphin Platform_ which appears to be discontinued. Rico is an (originally) API compatible fork the we continue to develop and improve.
+
+### Rationale
+
+[//]: # ( #### Motivation for Rico )
+
+The motivation for us to create Rico is to avoid code for standard use-cases to be written over-and-over again. Thus, we created it as an extension to commonly used frameworks like Spring and Jakarta EE (Java EE) that is easily adoptable.
```

Comment:
```suggestion
The motivation for us to create Rico is to avoid code for standard use-cases to be written over-and-over again. Thus, we created it as an extension to commonly used frameworks like many others that is easily adoptable.
```

Reply:
I think we should neame SPring and JEE explicitly here

Reply:
sorry, that was an misunderstanding.. You're right... I thought you have Rico compared with Spring/Jakarta EE :) Everything fine 👍